draft-ietf-quic-transport.md

```diff
@@ -5536,6 +5545,9 @@ The initial contents of this registry are shown in {{iana-tp-table}}.
 | 0x000e | active_connection_id_limit  | {{transport-parameter-definitions}} |
 {: #iana-tp-table title="Initial QUIC Transport Parameters Entries"}
 
+Additionally, each value of the format `31 * N + 27` for integer values of N
+(that is, `31`, `58`, `85`, ...) MUST NOT be assigned by IANA.
```

Comment:
```suggestion
(that is, `27`, `58`, `89`, ...) MUST NOT be assigned by IANA.
```
Your examples are of `31 + N * 27`, I believe....

Reply:
Thanks for catching this!